lib/Core/TxTree.cpp

```diff
@@ -2575,6 +2581,11 @@ void TxTreeNode::printTimeStat(std::stringstream &stream) {
          << ((double)getStoredCoreExpressionsTime.getValue()) / 1000 << "\n";
 }
 
+void TxTreeNode::printNodeCntStat(std::stringstream &stream) {
```

Comment:
I think this method is not necessary, as `TxTreeGraph::nodeCount` is accessible from `TxTree`.
I recommend moving the procedure in this method to `TxTree::getInterpolationStat()`, but do the printing at the start of the method instead of at the end.